src/libData/AccountData/Account.h

```diff
@@ -117,10 +117,13 @@ inline std::ostream& operator<<(std::ostream& out,
 class Account : public AccountBase {
   // The associated code for this account.
   bytes m_codeCache;
+  bytes m_initDataCache;
   Address m_address;  // used by contract account only
+  uint32_t m_scilla_version = -1;
```

Comment:
uint32_t cannot be -1 

Should it be `(uint32_t)-1` instead?

Reply:
Why not explicitly use something like `UINT32_MAX` ?